linode/manager
# fix: [M3-7822] -  Prevent Modification of Linode config 'interfaces' Array on No Changes

## Description 📝
This PR prevents modifying the value of a Linode config's 'interfaces': [] with defaults in the payload when there are no changes made. Refer to the ticket for more details.

## Target release date 🗓️
5/13

## Preview 📷
![image](https://github.com/linode/manager/assets/119517080/c436ce17-5822-4fd0-aa46-e243d503a0df)


## How to test 🧪

### Verification steps
(How to verify changes)
-  Create a linode.
- Navigate to linode config and click on edit button.
- Make no changes and click on save.
- Observe the payload, there shouldn't be any override's to the interfaces array with defaults. (ex: if interfaces: [] it should stay like that and not be modified)
- Ensure there is no regression for Add/ Edit and deleting configurations. 

## As an Author I have considered 🤔

*Check all that apply*
